view/adminhtml/web/lang/en.js

```diff
@@ -28,6 +29,9 @@
     "container.activationCheckCard.title": "Activation Check",
     "container.activationCheckCard.status": "Status: {status}",
     "container.activationCheckCard.success": "Your domain is now queued up to be rescanned. Please check back in a few hours.",
+    "container.advanceddos.title": "Advance DDoS",
+    "container.advanceddos.description": "Get protected from advance smart attacks",
+    "container.advanceddos.value": "Visit Cloudflare.com",
```

Comment:
It's annoying to update language folder for every plugin. Should we have a repo for language files?